web/impact/requirements/base.txt

```diff
@@ -53,4 +53,4 @@ django-oidc-provider==0.7.0
 django-sitetree==1.13.1
 django-graphql-jwt==0.1.10
 
-git+https://github.com/masschallenge/add2cal
+git+https://github.com/masschallenge/add2cal#AC-7728
```

Comment:
Presumably this branch tag will need to be removed after the AC-7728 branch on add2cal is merged, correct?

Reply:
Correct